src/pages/authentication/building-todo-app.md

```diff
@@ -145,7 +142,7 @@ export const appConfig = new AppConfig(['store_write', 'publish_data']);
 
 The `appDetails` and `userSession` objects are joined by the callback function
 [`finished`](https://github.com/blockstack/stacks-todos/blob/master/src/components/App.jsx#L31)
-in configuring Stacks Connect for authentication with the `authOptions` object:
+in configuring Stacks Auth for authentication with the `authOptions` object:
```

Comment:
Ditto here on the "Stacks authentication" instead of "Stacks Auth" question per above.